pypit/scripts/chk_edges.py

```diff
@@ -33,24 +33,20 @@ def parser(options=None):
 
 def main(pargs):
 
-    import os
-
     import pdb as debugger
     import time
 
-    from astropy.io import fits
-    from astropy.table import Table
-
     from pypit import ginga
+    from pypit import traceslits
     from pypit.arspecobj import get_slitid
 
     import subprocess
 
-    # List only?
-    trc_hdu = fits.open(pargs.file)
-    head0 = trc_hdu[0].header
-
-    mstrace = trc_hdu[0].data
+    # Load up
+    Tslits = traceslits.TraceSlits.from_master_files(pargs.root)
+    lordloc = Tslits.lcen
+    rordloc = Tslits.rcen
+    mstrace = Tslits.mstrace
```

Comment:
I'm not sure why this line is yellow.  I might have did something I didn't intend to...

Anyway, I think we should avoid statements like this.  Is it primarily for backwards compatibility?  Within a function, I think we just want to use the `Tslits` object, but we can pass `Tslits.lcen` between functions and call it `lordloc` within that called function.

Reply:
expunged